src/ts/secureMPC.ts

```diff
@@ -1,24 +1,44 @@
 import { EventEmitter } from "ee-typed";
 import type { IO } from "./types";
-import workerSrc from "./workerSrc.js";
-import nodeSecure2PC from "./nodeSecure2PC.js";
+import workerCode from "./workerCode.js";
+import nodeSecureMPC from "./nodeSecureMPC.js";
 
-export type Secure2PC = typeof secure2PC;
+export type SecureMPC = typeof secureMPC;
 
-export default function secure2PC(
-  party: 'alice' | 'bob',
+const getWorkerUrl = (() => {
+  let url: string | undefined;
+
+  return () => {
+    if (!url) {
+      const blob = new Blob([workerCode], { type: 'application/javascript' });
+      url = URL.createObjectURL(blob);
+    }
+
+    return url;
+  }
+})();
```

Comment:
This solves the problem Bun had with the base64 URL and web workers:  https://github.com/voltrevo/emp-wasm/pull/4 🦾  Deno is still complaining tho.. it needs the following code:

```ts
await replaceInFile(
  join(gitRoot, 'build/jslib.js'),
  ['import("module")'],                                                                                                                            
  'import("node:module")'                                                           
);
```
But then it throws another error related to the Worker URL parameter.  It could make sense to open another PR to solve this specifically tho.

Reply:
Awesome!

Curious about the Deno thing. Node+browser are working with this replacement so I included it in this PR. There's another error though yeah? Agreed to leave that for another PR 👍